lib/jnpr/junos/device.py

```diff
@@ -218,7 +218,6 @@ def _sshconf_lkup(self):
                 self._hostname = found.get('hostname', self._hostname)
                 self._port = found.get('port', self._port)
                 self._conf_auth_user = found.get('user')
-                self._conf_ssh_private_key_file = found.get('identityfile')
```

Comment:
Not sure why is this lined removed. 
So if I have entered my private ssh file as *identityfile* in my ssh cofig. witout this line those private key file will be ignored and its like everytime user need to pass with ssh_private_key_file parameter. 

In case of multiple device usage with different private file for different device. identityfile is of much use.